app/models/miq_template.rb

```diff
@@ -36,6 +36,19 @@ def self.without_volume_templates
                         "ManageIQ::Providers::Openstack::CloudManager::VolumeSnapshotTemplate"])
   end
 
+  def self.applied_filtering
+    tenant = User.current_user.current_group.tenant
+    if tenant.source_id
```

Comment:
PR works good for me (at least from OpenStack provider point of view).

Can we get MIQ-core eyes on this PR? @Ladas @agrare

Reply:
@lpichler hm, looks this this should be done via the tenant mapping? This looks like a wrong place.

@aufi Also I am confused about which tenant is in use here. Cause it looks like we use MIQ tenant, but we should rather use the mapped tenants? Or there should be 2 paths, with tenant mapping on&off?